packages/line/index.d.ts

```diff
@@ -185,6 +186,9 @@ export interface LineProps {
     onMouseMove?: PointMouseHandler
     onMouseLeave?: PointMouseHandler
     onClick?: PointMouseHandler
+    onTouchStart?: PointTouchHandler
```

Comment:
It would be nice to add these new properties to the website, but the way to do so is unfortunately not documented.

Reply:
I think we could also add tests for those, if possible.

Reply:
Added tests and docs!

Reply:
Thank you!